### PR TITLE
Add build tags for other 64 bit architectures

### PR DESCRIPTION
**Pull request by mwhudson**
As reported at https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=843787

### CODE REVIEW

**Comment by jacobsa** [comment]
Side note: it's a surprise to me that this package is included in debian. Do you have any idea how to tell who added it, why, or what it's being used for?

File: cmac/hash_64bit.go [hunk 1]
```diff
@@ -13,7 +13,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-// +build amd64 arm64
+// +build amd64 arm64 ppc64 ppc64le s390x mips64 mips64le
```
Comment (jacobsa) [inline cmac/hash_64bit.go]: The casts below assume that it's safe to perform unaligned word-sized loads. Are you sure that's true on all of these architectures? Could you please cite references in comments?

Comment (mwhudson) [inline cmac/hash_64bit.go]: Done.

**Comment by mwhudson** [comment]
It was only uploaded in October it seems: http://metadata.ftp-master.debian.org/changelogs/main/g/golang-github-jacobsa-crypto/golang-github-jacobsa-crypto_0.0~git20160410.0.42daa9d-2_changelog 

It seems it was uploaded as it is a dependency of gocryptfs (as in https://github.com/rfjakob/gocryptfs)

**Comment by jacobsa** [comment]
Thanks!

**Pull request merged**